sasl_bind.c

```diff
@@ -0,0 +1,21 @@
+#include <ldap.h>
+#include "sasl_bind.h"
+
+static int
+sasl_trivial_interact (LDAP * ld, unsigned flags, void *defaults,
+                       void *sasl_interact)
+{
+  (void) ld;
+  (void) flags;
+  (void) defaults;
+  (void) sasl_interact;
+  return LDAP_SUCCESS;
+}
+
+int
+trivial_external_sasl_bind (LDAP * ld)
+{
+  return ldap_sasl_interactive_bind_s (ld, NULL, "EXTERNAL", NULL, NULL,
```

Comment:
This calls authorization identity a DN: https://docs.oracle.com/middleware/11119/oid/develop/ext_ldap.htm#OIMAD2090

This says the server always ignores DN (quite wisely) https://linux.die.net/man/3/ldap_sasl_bins_s

Anyway, we should not be smarter than openldap is: clients do nothing in Sasl external auth. Neither should we :)